test/1_test_fullpath.js

```diff
@@ -977,10 +975,10 @@ contract('Voucher tests', async (addresses) => {
       );
 
       //Check VoucherKernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
-      assert.isTrue(voucherStatus.status.eq(new BN(194)));
+      assert.isTrue(voucherStatus[0].eq(new BN(194)));
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)